test/packages/good/manifest.yml

```diff
@@ -3,6 +3,7 @@ name: good
 title: Good package
 description: This package is good.
 version: 1.0.0
+license: platinum
```

Comment:
I suggest leaving the license of the `good` package as is, as many packages are using basic (default) and creating a new one for `platinum` (`elastic-package create`).

Reply:
Created a separate test package for platinum license in 0aa6454b67bd7d760b7dcc644e91050de50edaa2